### DISCUSSION TITLE
Self-nomination for contributor role

### DISCUSSION THREAD

**Discussion by robdimsdale** [paketo-buildpacks/feedback#112] [Dependencies Team]
I would like to self-nominate as a dependencies contributor. I have helped out on a variety of dependencies-related issues, including providing direction and feedback on the upcoming dependencies rewrite work. I intend on being involved in a lot of upcoming deps work, and being a contributor will make that easier for me and the rest of the dependencies team.

@paketo-buildpacks/dependencies-maintainers can I get a 👍 / 👎 ?

**Comment by ryanmoran** [comment]
Congrats @robdimsdale on becoming a Dependencies Contributor! 🎉
